### PR TITLE
mutt_ics: submission

### PR DESCRIPTION
**Pull request by gpanders**
#### Description

Add port for [mutt_ics](https://github.com/dmedvinsky/mutt-ics).

###### Tested on

macOS 10.15.7 19H524
Xcode 12.4 12D4e

###### Verification 
Have you

- [X] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [X] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [X] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [X] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [X] checked your Portfile with `port lint`?
- [ ] ~~tried existing tests with `sudo port test`?~~ N/A
- [X] tried a full install with `sudo port -vst install`?
- [X] tested basic functionality of all binary files?

### CODE REVIEW

File: mail/mutt_ics/Portfile [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+homepage            https://github.com/dmedvinsky/mutt-ics
+maintainers         {@gpanders gpanders.com:greg} \
+                    openmaintainer
+
+description         A tool to show calendar event details in Mutt
+long_description    mutt_ics allows you to view calendar attachments inline \
+                    in Mutt.
+
+python.default_version \
+                    38
```
Comment (herbygillot) [inline mail/mutt_ics/Portfile:20]: We're defaulting to Python 3.9 these days - any particular reason for 3.8?

Comment (gpanders) [reply mail/mutt_ics/Portfile:20]: Only that `py-icalendar` only supports up to 3.8. I can update the icalendar port as well and include that in this PR and have them both use 3.9.

Comment (gpanders) [reply mail/mutt_ics/Portfile]: Updated py-icalendar and mutt_ics to use Python 3.9

**Review by herbygillot — APPROVED** [review]

**Comment by herbygillot** [comment]
Lgtm. Merged.

**Pull request merged**